doc/profiler.rst

```diff
@@ -0,0 +1,56 @@
+########
+Profiler
+########
+
+The **Profiler** pane recursively determines the run time and number of calls for every function and method called in a script, whether directly or indirectly, breaking down each procedure into its smallest individual units.
```

Comment:
Let's change `script` for `file` here, which is easier to understand for foreign users.

Reply:
Sure, if you think so. Done.